active_learning_multiannotator/active_learning.ipynb

```diff
@@ -0,0 +1,677 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.plot(model_accuacy_arr)\n",
+    "plt.xticks(range(num_rounds))\n",
+    "plt.xlabel(\"Round\")\n",
+    "plt.ylabel(\"Model Accuracy\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the plot above, we can see that the model accuracy increases steadily with each additional round of collecting more labels, getting improved consensus labels, and model training."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Other useful multiannotator statistics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After conducting multiple rounds of active learning, you can also obtain various multiannotator statistics such as the higher quality [CROWDLAB consensus label](https://arxiv.org/abs/2210.06812) for each example, consensus label quality scores, annotator quality scores and more for the examples that have been labeled. \n",
+    "\n",
+    "Below we show a simple example of how to obtain this information using the annotators' labels and the model predicted probabilites. Check out cleanlab's [multiannotator module](https://docs.cleanlab.ai/stable/tutorials/multiannotator.html) for more detailed information!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# using the annotator labels and predicted probabilites to get the consensus labels and other statistics\n",
```

Comment:
for these one line commands, my personal pref is to move the comment above out of the code cell and have it be markdown text describing what the single line of code does.